pimd/pim6_cmd.c

```diff
@@ -1913,12 +1914,12 @@ DEFPY (show_ipv6_pim_cand_rp,
 
 DEFPY (show_ipv6_pim_bsr_rpdb,
        show_ipv6_pim_bsr_rpdb_cmd,
-       "show ipv6 pim bsr candidate-rps [vrf VRF_NAME] [json$uj]",
```

Comment:
again this command is just gone.  We need to provide an upgrade path and cannot just remove commands

Reply:
same comment above.